### PR TITLE
Logo update

### PR DESCRIPTION
**Pull request by kststudio**
Replaces the logo with a new logo that is using the correct shade of purple

### CODE REVIEW

**Comment by kststudio** [comment]
changes cursor from text to default and removes comments from css file

File: src/components/TopBar/logo.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import logo from "../../assets/icons/msm_logo.png";
 
 export default class Logo extends React.Component {
   render() {
     return (
         <div id="logo">
-            <img src={logo}
-            alt="Migrate Service Map"
-            />
+            <a href="https://refugeeswelcomehome.org/">
```
Comment (alexjball) [inline src/components/TopBar/logo.js]: Could you add target="_blank" to this element? That make the link open in a new tab. The map does not store saved items between page loads so we don't want to lose progress if the user clicks on the logo.

**Review by alexjball — APPROVED** [review]
LGTM aside from comments!

File: src/components/TopBar/logo.js (hunk 1 above)
Comment (kststudio) [reply src/components/TopBar/logo.js]: @alexjball This has been added in commit #8c14aab

**Pull request merged**